Dockerfile

```diff
@@ -3,6 +3,9 @@
 
 FROM docker.io/library/golang:1.21.0-alpine as builder
 
+ARG OPI_API_VERSION
+LABEL opi-api-version=${OPI_API_VERSION}
```

Comment:
Do we still need this part, since we do not set the version as a build argument in CI?

Reply:
No we do not need it, updated this PR with deleting the ARG and LABEL in Dockerfile.

Tested in https://github.com/yanjing1104/opi-spdk-bridge/tree/test2 

```bash
# docker pull ghcr.io/yanjing1104/opi-spdk-bridge:pr-2
pr-2: Pulling from yanjing1104/opi-spdk-bridge
7264a8db6415: Already exists 
6b2082003f5c: Pull complete 
e9d6d61eb267: Pull complete 
Digest: sha256:4f4c8e36261efe0f9f2a14d7c16edf191234c96273b15077af3afefb09439d74
Status: Downloaded newer image for ghcr.io/yanjing1104/opi-spdk-bridge:pr-2
ghcr.io/yanjing1104/opi-spdk-bridge:pr-2

# docker image ls | head -n2
REPOSITORY                                         TAG             IMAGE ID       CREATED             SIZE
ghcr.io/yanjing1104/opi-spdk-bridge                pr-2            b4eb501e442f   12 minutes ago      52.4MB

# docker inspect b4eb501e442f --format '{{ json .Config.Labels }}' | sed 's/,/\n/g'
{"opi-api-version":"v0.0.0-20230826011814-48b273e85a4f"
"org.opencontainers.image.created":"2023-08-29T08:33:19.060Z"
"org.opencontainers.image.description":"OPI Storage gRPC to SPDK json-rpc bridge"
"org.opencontainers.image.licenses":"Apache-2.0
"
"org.opencontainers.image.revision":"d8d96d366dafb03bd0d0cf72d46b4811de328abc"
"org.opencontainers.image.source":"https://github.com/yanjing1104/opi-spdk-bridge"
"org.opencontainers.image.title":"opi-spdk-bridge"
"org.opencontainers.image.url":"https://github.com/yanjing1104/opi-spdk-bridge"
"org.opencontainers.image.version":"pr-2"}
```